### ISSUE TITLE
Support localization

### ISSUE THREAD

**Issue by dademaru** [hallme/woo-address-book#76]
Since the translation of this plugin on the official repository seems to take too long (suggested Italian translation hasn't been approved for 7 month), could you please add **localization support** as described here?
https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#loading-text-domain
```
add_action( 'plugins_loaded', 'wc_address_book_load_plugin_textdomain' );
function wc_address_book_load_plugin_textdomain() {
    load_plugin_textdomain( 'woo-address-book, FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
}
```
I've just added Italian translation on WP repository, but meanwhile I would translate it with **PoEdit** and use my translations in my project.

[EDIT]
Sorry I haven't read the README, the translation FAQs.
I would apply as a translation editor as soon as I can.
Anyway, I think that adding `load_plugin_textdomain` is still a valid alternative since **PoEdit** requires it and returns this error otherwise:
> no load_plugin_textdomain call and lacking metadata
> some work is probably required to make the PHP code localization-ready
> see https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/

Thanks

**Comment by matt-h** [comment]
Hi,
`load_plugin_textdomain` should no longer be needed at all since WordPress 4.6+

I'm not sure where you are getting that specific error in PoEdit.

To use PoEdit, you should be able to just go here:
https://translate.wordpress.org/projects/wp-plugins/woo-address-book/stable/it/default/

At the bottom by "Export", select "only matching the filter" and "Portable Object Message Catalog (.po/.pot)" and then click "Export"

That will download a file name `wp-plugins-woo-address-book-stable-it.po` which you can open up in PoEdit.

Once it looks good in there select the option to "Compile to MO" and save the `.mo` file as `/wp-content/languages/plugins/woo-address-book-it_IT.mo`. This will let WordPress use the translation file automatically without adding anything to the plugin directory.
If the `/wp-content/languages/plugins` directory doesn't exist yet; go to the WordPress Updates section in your WordPress admin and click the "Update Translations" button at the bottom.

**Comment by dademaru** [comment]
Hello @matt-h 

I know it's deprecated since WP 4.6+ as reported in docs, but I was used to have `load_plugin_textdomain` in almost every plugin ([even WC uses it](https://github.com/woocommerce/woocommerce/blob/master/includes/class-woocommerce.php#L593)) and I think it's still a valid code to let users translate locally or use custom translations and keep them updated to plugin code.

Concerning PoEdit error, I got it when I tried to add a new translation for your plugin locally with PoEdit Pro 2.3.

Anyway, I did as you indicated.

Thanks
